custom_report_label/models/custom_report_label.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2024 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class CustomReportLabel(models.Model):
+    _name = "custom.report.label"
+
+    name = fields.Char(required=True)
+    model_id = fields.Selection(
+        selection="_list_all_models", string="Model", required=True
+    )
+    template_id = fields.Many2one("ir.actions.report")
```

Comment:
Is it would be possible to apply the template specification to all templates, not just ir.actions.report.

As background, we want to be able to resolve to requests to change the display of a website with this module.